app/src/androidTest/java/tech/ula/model/MigrationTest.kt

```diff
@@ -94,6 +94,8 @@ class MigrationTest {
         assertEquals(session2, returnedSession2)
         assertFalse(returnedSession1.isAppsSession)
         assertTrue(returnedSession2.isAppsSession)
+        assert(fs.defaultVncPassword == "userland")
```

Comment:
`assert` is actually disabled on the JVM by default. Tests have to be run with special flags in order to get them to fire and I can't remember if that's setup right now. For safeties sake, use stuff like `assertEquals` from Junit.